crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo

```diff
@@ -486,81 +491,1014 @@ mod YASPoolTests {
+                    );
+
+                    mint_callback
+                        .mint(
+                            yas_pool.contract_address,
+                            WALLET(),
+                            IntegerTrait::<i32>::new(240, true),
+                            yas_pool.tick_spacing(),
+                            150
+                        );
+
+                    assert(
+                        yas_pool.ticks(IntegerTrait::<i32>::new(240, true)).liquidity_gross == 250,
+                        'wrong liquidity_gross amount 5'
+                    );
+                    assert(
+                        yas_pool.ticks(IntegerTrait::<i32>::new(0, false)).liquidity_gross == 100,
+                        'wrong liquidity_gross amount 6'
+                    );
+                    assert(
+                        yas_pool.ticks(yas_pool.tick_spacing()).liquidity_gross == 150,
+                        'wrong liquidity_gross amount 7'
+                    );
+                    assert(
+                        yas_pool
+                            .ticks(
+                                IntegerTrait::<i32>::new(tick_spacing(FeeAmount::MEDIUM) * 2, false)
+                            )
+                            .liquidity_gross == 0,
+                        'wrong liquidity_gross amount 8'
+                    );
+
+                    mint_callback
+                        .mint(
+                            yas_pool.contract_address,
+                            WALLET(),
+                            IntegerTrait::<i32>::new(0, false),
+                            IntegerTrait::<i32>::new(tick_spacing(FeeAmount::MEDIUM) * 2, false),
+                            60
+                        );
+
+                    assert(
+                        yas_pool.ticks(IntegerTrait::<i32>::new(240, true)).liquidity_gross == 250,
+                        'wrong liquidity_gross amount 9'
+                    );
+                    assert(
+                        yas_pool.ticks(IntegerTrait::<i32>::new(0, false)).liquidity_gross == 160,
+                        'wrong liquidity_gross amount 10'
+                    );
+                    assert(
+                        yas_pool.ticks(yas_pool.tick_spacing()).liquidity_gross == 150,
+                        'wrong liquidity_gross amount 11'
+                    );
+                    assert(
+                        yas_pool
+                            .ticks(
+                                IntegerTrait::<i32>::new(tick_spacing(FeeAmount::MEDIUM) * 2, false)
+                            )
+                            .liquidity_gross == 60,
+                        'wrong liquidity_gross amount 12'
+                    );
+                }
+
+
+                // TODO: 'removes liquidity from liquidityGross'
+                //#[test]
+                //#[available_gas(200000000)]
+                fn test_remove_liquidityGross() {
```

Comment:
Please comment this entire function leaving a comment saying that it's missing the burn functionality.

Reply:
done